libs/application/templates/payable-dummy-template/README.md

```diff
@@ -0,0 +1,67 @@
+# Payable Template
+
+4. Add to `application/template-loader/src/lib/templateLoaders.ts` so that library knows how to import this new application template.
```

Comment:
what are steps 1 through 3?

Reply:
@sindrig This is a stand-alone application - the sole purpose of the dummy template is to allow individuals from various companies to test out the payment function of the application, without going through the application it self.
Regarding the three steps, if you're interested, maybe @arnorhs can answer that.